validator/validator-main.protoascii

```diff
@@ -3492,6 +3492,28 @@ tags: {
   }
 }
 
+# Dynamic Viewport (dvh) Polyfill when amp-story-1.0.css is Server Side Rendered.
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "SCRIPT"
+  spec_name: "script amp-story-dvh-polyfill"
+  descriptive_name: "script amp-story-dvh-polyfill"
+  unique: true
+  mandatory_parent: "HEAD"
+  attrs: {
+    name: "amp-story-dvh-polyfill"
+    value: ""
+    mandatory: true
+    dispatch_key: NAME_VALUE_DISPATCH
+  }
+  cdata: {
+    # Only valid if contents are exactly either of these:
```

Comment:
Remove "either of"? I only see one option.

Reply:
will follow up! ty